internal/adapters/email/smtp_sender.go

```diff
@@ -0,0 +1,47 @@
+package email
+
+import (
+	"bytes"
+	"path/filepath"
+	"text/template"
+
+	"github.com/ezex-io/ezex-notification/config"
+	"github.com/ezex-io/ezex-notification/internal/ports"
+	"gopkg.in/gomail.v2"
+)
+
+type SMTPAdapter struct {
+	config *config.Config
+}
+
+func NewSMTPAdapter(cfg *config.Config) ports.EmailSender {
+	return &SMTPAdapter{config: cfg}
+}
+
+func (a *SMTPAdapter) SendOTPEmail(to, otp string) error {
+	tmplPath := filepath.Join("assets", "template", "email", "otp.md")
```

Comment:
For prevent memory leak init template object into `map[string]*template.Trmplate`

Key is file name.

Before use, embed fs for templates.